README.md

```diff
@@ -153,8 +153,10 @@ I would like to thank Microverse
 
 <!-- LICENSE -->
 
-## 📝 License <a id="license"></a>
+## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please fix the link to this license because it leads to a 404 page.
![image](https://github.com/theresetuyi/hello-rails-react/assets/45460192/9e371667-7f97-47db-bce0-68443c362bdd)